source/command-monitoring/command-monitoring.rst

```diff
@@ -392,6 +406,13 @@ See the `Load Balancer Specification <../load-balancers/load-balancers.rst#event
      */
     connectionId: ConnectionId;
 
+    /**
+     * Returns the server connection id for the command. The server connection id is distinct from
+     * the connection id and is returned by the hello or legacy hello response as "connectionId"
+     * from the server on 4.2+.
+     */
+    serverConnectionId: Optional<Int64>;
```

Comment:
Why is this the only case where `serverConnectionId` is optional?

Reply:
Typo... added `Optional` to all.